tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1769,6 +1788,24 @@ schedule(void)
             i->misc_metadata = loop_md;
           }
         }
+
+        if (BIH_VECTORLENGTH_ENABLED(bih)) {
+          if (LL_MDREF_IS_NULL(loop_md)) {
+            loop_md = cons_loop_id_md();
+          }
+          LL_MDRef vectorlength_enable = cons_vectorize_metadata();
+          ll_extend_md_node(cpu_llvm_module, loop_md, vectorlength_enable);
+          LL_MDRef vectorlength_scalable  = cons_vectorlength_scalable_metadata(BIH_VECTORLENGTH_SCALABLE( bih )>0);
+          ll_extend_md_node(cpu_llvm_module, loop_md, vectorlength_scalable);
+          if (vectorlength_factor > 0) {
+            LL_MDRef lvcomp[2];
```

Comment:
These 5 lines should be outlined into a constructor function, as with the prevailing style.